apps/logging/loggers.py

```diff
@@ -387,6 +390,7 @@ def log_global_state_metrics(group_timestamp=None, report_flag=True):
             "user_date_joined": format_timestamp(app.user.date_joined),
             "user_last_login": format_timestamp(app.user.last_login),
             "user_organization": getattr(user_profile, "organization_name", None),
+            "user_limit_data_access": user_limit_data_access
```

Comment:
@sharonfruit To get the current type for the applications, can you add a similar line as the above.  Would just be something like:

```
     "data_access_type":  app.data_access_type
```

The related tests are under:  apps/logging/tests/test_loggers_management_command.py

Reply:
This actually already exists on line 356

Reply:
@sharonfruit Sorry I missed that.  Looks like it was already added in the past. 

I wasn't finding any related tests for "data_access_type". We should add it as a required field in the tests.

Reply:
@dtisza1 No problem, definitely adding to the existing test.